### PR TITLE
fix: add missing widht and height attributes for images

### PR DESCRIPTION
**Pull request by dawid-ziobro**
### 🔗 Linked issue

https://vsf.atlassian.net/browse/SV-91

### ❓ Type of change

- [ ] 📖 Documentation (updates to the documentation, readme or JSDoc annotations)
- [ ] 🐞 Bug fix (a non-breaking change that fixes an issue)
- [ ] 👌 Enhancement (improving an existing functionality like performance)
- [ ] ✨ New feature (a non-breaking change that adds functionality)
- [ ] 🧹 Chore (updates to the build process or auxiliary tools and libraries)
- [ ] ⚠️ Breaking change (fix or feature that would cause existing functionality to change)

### 📚 Description

Fixes for best practices and CLS.

### 📝 Checklist

- [x] I have linked an issue or discussion.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: apps/web/components/ui/Hero/Hero.vue [hunk 1]
```diff
@@ -2,7 +2,7 @@
   <div class="relative min-h-[600px] mb-10">
     <div class="md:flex md:flex-row-reverse md:justify-center min-h-[600px] max-w-screen-3xl mx-auto">
       <div class="flex flex-col justify-center md:basis-2/4 md:items-stretch md:overflow-hidden">
-        <NuxtImg :src="image" alt="Hero" class="h-auto w-full object-cover object-left" height="600" width="764" />
+        <NuxtImg :src="image" alt="Hero" class="h-auto w-full object-cover object-left" width="764" height="600" />
```
Comment (FRSgit) [inline apps/web/components/ui/Hero/Hero.vue:5]: 😄

Comment (FRSgit) [reply apps/web/components/ui/Hero/Hero.vue:5]: Why the order change? 😄

**Review by FRSgit — APPROVED** [review]
Looks good, though I do not understand why the order is changed 😄  Do you think we should force attributes order and have some eslint rule about that?

File: apps/web/components/ui/Hero/Hero.vue (hunk 1 above)
Comment (dawid-ziobro) [reply apps/web/components/ui/Hero/Hero.vue:5]: Chasing the perfection 😄

**Comment by dawid-ziobro** [comment]
@FRSgit naah, i'm far away for now from such solutions. I've done it since it is a small change but my eyes were bleeding seeing such inconsistency 😄

**Pull request merged**